app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitListFragment.java

```diff
@@ -76,7 +76,14 @@ public View onCreateView(@NonNull LayoutInflater inflater,
                 String title = (String) extractMap.get("title");
                 String reason = (String) extractMap.get("reason");
                 Timestamp date = (Timestamp) extractMap.get("date");
-                Long id = (Long) extractMap.get("habitId");
+                Long id;
+                // TODO: This is a temp fix to allow both versions in the db to work
```

Comment:
I will raise a task to fix this in about a week so that there is a settling period for everyone.